sample-app/src/main/java/com/squareup/contour/sample/ExpandableBioCard1.kt

```diff
@@ -33,10 +34,15 @@ import com.squareup.contour.ContourLayout
 import com.squareup.picasso.Picasso
 
 @SuppressLint("SetTextI18n")
-class ExpandableBioCard1(context: Context) : ContourLayout(context) {
+class ExpandableBioCard1(context: Context, attributeSet: AttributeSet? = null) : ContourLayout(context) {
   private val avatar = CircularImageView(context).apply {
     scaleType = CENTER_CROP
-    Picasso.get().load("https://upload.wikimedia.org/wikipedia/en/9/92/BenSisko.jpg").into(this)
+
+    //image will appear on emulator and devices
```

Comment:
This comment isn't necessary, the code is self-documenting.

Reply:
Same for the comment in `ExpandableBioCard2`

Reply:
done